pyproject.toml

```diff
@@ -33,6 +33,18 @@ dependencies = [
 ]
 dynamic = ["version"]
 
+[project.optional-dependencies]
+test = [
+    "coverage",
+    "pytest",
+]
+
+optional = [
+    "streamlit",
+    "boto3",
+    "moto[s3]",
```

Comment:
`moto[s3]` is a mock library for boto3, and is used by unit tests only. How about moving to the `test` category?

Reply:
Fixed.